util/util.go

```diff
@@ -89,6 +93,7 @@ func ResolveName(name string, firstRegistry string, sc *types.SystemContext, sto
 		}
 	}
 
+	name = strings.TrimPrefix(name, DefaultTransport)
```

Comment:
In my review I'd done this at one point, but was concerned that the DefaultTransport might not be the only valid transport at some point and to trust the user.  Was I over thinking?  This did work when I tested it though.

Reply:
Well the Default Transport is the only one that looks like `NAME://`  All other transports are `NAME:` and are handled separately.

So I don't see this as an issue.

Reply:
Right, but this code only works for "docker://", would it be valid to have something else there at some point like "oci://"?  Also if they put in "Docker://" they'll get a bad reference message which is probably OK, not sure if you want to lowercase name or not.